.travis.yml

```diff
@@ -40,6 +40,7 @@ install:
 
 script:
   - bundle exec rake lint
+  - rake ci:run[mongo]
   - rake ci:run[default]
```

Comment:
now that the new gem with your fixes has been released, can we put [default] back as the first rake task?